### PR TITLE
Kf1.7 i18n

### PR DESCRIPTION
**Pull request by mathis-marcotte**
closes https://github.com/StatCan/jupyter-apis/issues/221

### CODE REVIEW

File: frontend/jupyter/src/app/pages/form/form-new/form-protected-b/form-protected-b.component.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
-<lib-form-section title="Protected B">
+<lib-form-section title="Protected B" i18n-title>
   <div class="row">
     <div class="flex-column">
-      <mat-checkbox [formControl]="parentForm.get('prob')" i18n>
+      <mat-checkbox [formControl]="parentForm.get('prob')">
         <mat-icon matTooltip="Protected B" i18n-matTooltip> lock </mat-icon>
-        Run a Protected B notebook
+        <span i18n>&nbsp;Run a Protected B notebook</span>
```
Comment (wg102) [inline frontend/jupyter/src/app/pages/form/form-new/form-protected-b/form-protected-b.component.html:6]: Why the non-breaking space?

File: frontend/jupyter/src/app/pages/form/form-new/form-image/form-image.component.html [hunk 2]
```diff
@@ -76,7 +76,7 @@
       >
         <mat-icon class="server-type-icon" svgIcon="group-three"></mat-icon>
         <p class="server-type-title">SAS</p>
-        <p class="server-type-content">This is text for SAS</p>
+        <p class="server-type-content" i18n>A SAS image.</p>
```
Comment (wg102) [inline frontend/jupyter/src/app/pages/form/form-new/form-image/form-image.component.html:79]: Do we want a better name? Like A SAS image that allows to xyz?

File: frontend/jupyter/src/app/pages/form/form-new/form-image/form-image.component.html [hunk 3]
```diff
@@ -56,8 +56,8 @@
         "
       >
         <mat-icon class="server-type-icon" svgIcon="group-two"></mat-icon>
-        <p class="server-type-title">Remote Desktop</p>
-        <p class="server-type-content">A ubuntu remote desktop.</p>
+        <p class="server-type-title">Ubuntu</p>
+        <p class="server-type-content" i18n>A ubuntu remote desktop.</p>
```
Comment (wg102) [inline frontend/jupyter/src/app/pages/form/form-new/form-image/form-image.component.html:60]: An Ubuntu remote desktop?

File: frontend/jupyter/src/app/pages/volume-form/volume-form.component.html [hunk 4]
```diff
@@ -52,9 +52,9 @@
     <mat-form-field appearance="outline" class="wide">
       <mat-label i18n>Access Mode</mat-label>
       <mat-select formControlName="mode">
-        <mat-option value="ReadWriteOnce" i18n>ReadWriteOnce</mat-option>
-        <mat-option value="ReadOnlyMany" i18n>ReadOnlyMany</mat-option>
-        <mat-option value="ReadWriteMany" i18n>ReadWriteMany</mat-option>
+        <mat-option value="ReadWriteOnce">ReadWriteOnce</mat-option>
+        <mat-option value="ReadOnlyMany">ReadOnlyMany</mat-option>
+        <mat-option value="ReadWriteMany">ReadWriteMany</mat-option>
```
Comment (wg102) [inline frontend/jupyter/src/app/pages/volume-form/volume-form.component.html:57]: Why remove the translations for those? Unless they were never really translated, but it was done in a 'eventually we will need good translation' idea.

File: frontend/jupyter/src/app/services/config.ts [hunk 5]
```diff
@@ -19,7 +19,7 @@ export function getStopDialogConfig(name: string): DialogConfig {
   return {
     title: $localize`Are you sure you want to stop this notebook server? ${name}`,
     message: $localize`Warning: Your data might be lost if the notebook server
-                       is not backed by persistent storage.`,
+                       is not backed by persistent storage`,
```
Comment (wg102) [inline frontend/jupyter/src/app/services/config.ts:22]: Why remove the . ?

**Review by wg102 — CHANGES_REQUESTED** [review]

**Review by wg102 — CHANGES_REQUESTED** [review]
Adding to the code review:
- [ ] Entrer le nom d'une propriété ou d'une valeur
image
- [ ] Untranslated text for the help 
image
- [ ] Untranslated Log out (might be kubeflow) 
image
- [ ] Untranslated value in table (might not be possible to translate) 
image
- [ ] Untranslated Help text 
image

File: frontend/jupyter/src/app/pages/form/form-new/form-protected-b/form-protected-b.component.html (hunk 1 above)
Comment (mathis-marcotte) [reply frontend/jupyter/src/app/pages/form/form-new/form-protected-b/form-protected-b.component.html:6]: Styling reason. With the span surrounding the text for easy i18n declaration, there was no space between the icon and the text. The non breaking space is to replicate how it looked like before surrounding the text with the span.

File: frontend/jupyter/src/app/pages/form/form-new/form-image/form-image.component.html (hunk 2 above)
Comment (mathis-marcotte) [reply frontend/jupyter/src/app/pages/form/form-new/form-image/form-image.component.html:79]: Better name, I don't think so. Better text description, yes. But I really don't know what text should be there.

File: frontend/jupyter/src/app/pages/form/form-new/form-image/form-image.component.html (hunk 3 above)
Comment (mathis-marcotte) [reply frontend/jupyter/src/app/pages/form/form-new/form-image/form-image.component.html:60]: That does seem better with english rules in mind of "an" before vowels. Good catch

File: frontend/jupyter/src/app/services/config.ts (hunk 5 above)
Comment (mathis-marcotte) [reply frontend/jupyter/src/app/services/config.ts:22]: The getDeleteDialogConfig didn't have a dot at the end of the warning message, so I was just making it consitent. Unless you think the solution is to add the dot to the other warning messages.

File: frontend/jupyter/src/app/pages/form/form-new/form-image/form-image.component.html (hunk 2 above)
Comment (wg102) [reply frontend/jupyter/src/app/pages/form/form-new/form-image/form-image.component.html:79]: Yes description is what I meant. But at worst, it`s already better than what was there before.

**Review by wg102 — CHANGES_REQUESTED** [review]
A few more changes 
image
- [ ] Example: This is English. Exemple is the French word.
- [ ] Le caractère "-" ]
- [x] soit "au bloc-notes" soit "aux bloc-notes" => Dans ce cas c'est correcte

**Review by wg102 — APPROVED** [review]

**Pull request merged**